src/plugins/melpa-resolver.js

```diff
@@ -0,0 +1,82 @@
+const Joi = require('joi');
+const findReachableUrls = require('find-reachable-urls');
+const got = require('got');
+const insight = require('../utils/insight');
+
+let lastModified = 'Mon, 19 Jun 2017 15:27:49 GMT';
```

Comment:
Yeah, I had tried just leaving it undefined (`let lastModified;`), but then it didn't work. I just pushed up a fix that removes the hardcoded date: 637aed6f9d2e13dda557b49422a42611f8c90a0a